Kentico/kontent-sample-app-vue
# refactor(configuration): update layout to match new ui

### Motivation

Which issue does this fix? Fixes #82

### Checklist


### How to test

If manual testing is required, what are the steps?

Make sure the new UI is identical to [https://github.com/Kentico/kontent-sample-app-net/pull/128](https://github.com/Kentico/kontent-sample-app-net/pull/128)
